src/borg/helpers/parseformat.py

```diff
@@ -472,16 +472,6 @@ def normpath_special(p):
             p = os.path.normpath(p)
             return ("/." + p) if relative else p
 
-        if is_win32:
-            m = self.win_file_re.match(text)
```

Comment:
what's about self.win_file_re now? do we keep it for later?

Reply:
yes, it will be useful to fix the remote tests.

Reply:
ok.